File: Java/Day06_Invertedpyramidrotate.java

/**
 * Day 06: Inverted Half Pyramid (Rotated 180 deg)
 * Printing a triangle of stars aligned to the right.
 */
import java.util.*;

public class Day06_Invertedpyramidrotate {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the value of n:");
        int n = sc.nextInt();

        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= n - i; j++) {
                System.out.print(" ");
            }
            for (int j = 1; j <= i; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: Python/Day06_Invertedpyramidrotate.py
"""
Day 06: Inverted Half Pyramid (Rotated 180 deg)
Printing a triangle of stars aligned to the right.
"""

n = int(input("Enter the value of n:"))

for i in range(1, n + 1):
    # Print spaces
    for j in range(1, n - i + 1):
        print(" ", end="")
    # Print stars
    for j in range(1, i + 1):
        print("*", end="")
    print()
